### PR TITLE
Elasticsearch setup analytics

### PR DESCRIPTION
**Pull request by djordje-mijatovic**
## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

This will fix the following error when specifying DATAHUB_ANALYTICS_ENABLED=false:

> going to use protocol: https
> going to use elastic headers based on username and password
> going to use prefix: 'datahub-abc_'
> 
> datahub_analytics_enabled: false
> 2022/10/26 13:42:46 Command exited with error: exit status 1

### CODE REVIEW

**Comment by mattmatravers** [comment]
This should probably be committed as `fix(elasticsearch): elasticsearch-setup-job fails when DATAHUB_ANALYTICS_ENABLED=false`

**Review by mattmatravers — APPROVED** [review]
Looks great - I've tested this locally and it fixes the problem

File: docker/elasticsearch-setup/create-indices.sh [hunk 1]
```diff
@@ -148,11 +148,11 @@ if [[ $DATAHUB_ANALYTICS_ENABLED == true ]]; then
   fi
 else
   echo -e "\ndatahub_analytics_enabled: $DATAHUB_ANALYTICS_ENABLED"
-  DATAHUB_USAGE_EVENT_INDEX_RESPONSE_CODE=$(curl -o /dev/null -s -w "%{http_code}" --header "$ELASTICSEARCH_AUTH_HEADER" "${ELASTICSEARCH_INSECURE}$ELASTICSEARCH_PROTOCOL://$ELASTICSEARCH_HOST:$ELASTICSEARCH_PORT/cat/indices/${PREFIX}datahub_usage_event")
+  DATAHUB_USAGE_EVENT_INDEX_RESPONSE_CODE=$(curl "${CURL_ARGS[@]}" -o /dev/null -w "%{http_code}" "$ELASTICSEARCH_URL/cat/indices/${PREFIX}datahub_usage_event")
```
Comment (mattmatravers) [inline docker/elasticsearch-setup/create-indices.sh:151]:
```suggestion
  DATAHUB_USAGE_EVENT_INDEX_RESPONSE_CODE=$(curl "${CURL_ARGS[@]}" -o /dev/null -w "%{http_code}" "$ELASTICSEARCH_URL/_cat/indices/${PREFIX}datahub_usage_event")
```

As per https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-indices.html, the correct url is `_cat/indices/`

**Review by mattmatravers — CHANGES_REQUESTED** [review]
Whoops - one character fix!

**Comment by mattmatravers** [comment]
For context, this first broke in https://github.com/datahub-project/datahub/pull/5502

**Review by mattmatravers — APPROVED** [review]
Looks good and I've tested the code paths locally

**Review by david-leifker — APPROVED** [review]

**Review by jjoyce0510 — APPROVED** [review]

**Pull request merged**